content/strategy-goals/strategy/growth-team/index.md

```diff
@@ -39,16 +51,20 @@ Our overall strategy is to unlock the four key themes that will enable product-l
 
 <iframe src="https://docs.google.com/presentation/d/e/2PACX-1vThxOgLUUK2EfBhYRoCPwfKcw8wiarmD7uaTtmQpB1_WL7oho1377pRT8Vv6l7avKtCdK4T1o9Qaqng/embed?start=false&loop=false&delayms=3000&slide=id.g1581317b2af_2_0" allow="fullscreen" title="Product Led Growth"></iframe>
 
-In the coming weeks (September and early October), we are focusing on [creating a self-service signup flow](https://docs.google.com/document/d/1aUfXlt5AGwhG7tIF8dPRmsLhFL8TuvPKFvXlOsxgFws) in support of the cloud launch . After that, we want to work on up to four threads in parallel:
-
-- **Iterate on the signup experience.** Build a fast signup experience that feels like a product.
-- **Improve admin onboarding and instance configuration.**
-- **Refine user activation metrics and iterate on user onboarding and activation.**
-- **Explore feedback loops and collaboration.** The scope of this is TBD. The first step is for product/design to research the key feedback loops/collaboration moments in the current Sourcegraph experience, and to make sure they result in acquisition/activation.
+Now that we have launched an initial, semi-automated signup flow, our initial priorities are:
 
-Here's a (private) [roadmap proposal](https://docs.google.com/document/d/1HSSHT_qW3f6unEl3VXZNZi9Uw13wX-l9e0LYJEKOrdY/edit#) that will be merged in here once reviewed.
+- **Improve initial user onboarding**, in particular (a) being able to discover the value of Sourcegraph and (b) adding code. To do so, we're working on an onboarding widget ([#43596](https://github.com/sourcegraph/sourcegraph/issues/43596)), and investigating how to make adding code one-click (not started yet).
```

Comment:
nit: should "(a)" and "(b)" be swapped? To me, current (a) depends on (b).

Reply:
There's no dependency eg. if we add OSS code by default, _but_ the swap makes logical sense. Done